README.md

```diff
@@ -123,15 +123,15 @@ Google Cloud BigQuery (Alpha)
 
 Here is a code snippet showing a simple usage example from within Compute/App Engine. Note that you
 must [supply credentials](#authentication) and a project ID if running this snippet elsewhere.
+Complete source code can be found at
+[gcloud-java-examples:com.google.gcloud.examples.bigquery.snippets.CreateTableAndLoadData](https://github.com/GoogleCloudPlatform/gcloud-java/tree/master/gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/snippets/CreateTableAndLoadData.java).
```

Comment:
Can we use [relative link](https://help.github.com/articles/relative-links-in-readmes/)?

Also, "gcloud-java-examples:com.google.gcloud.examples.bigquery.snippets.CreateTableAndLoadData" is long and does not look great...